test/conformance/api/shared/util.go

```diff
@@ -36,6 +37,13 @@ import (
 
 const scaleToZeroGracePeriod = 30 * time.Second
 
+// DigestResolutionExceptions holds the set of "registry" domains for which
+// digest resolution is not required.  These "registry" domains are generally
+// associated with images that aren't actually published to a registry, but
+// side-loaded into the cluster's container daemon via an operation like
+// `docker load` or `kind load`.
+var DigestResolutionExceptions = sets.NewString("kind.local", "ko.local", "dev.local")
```

Comment:
It's a shame we can't read those from config because this is a conformance test :/